packages/model-data-generator/src/references.js

```diff
@@ -21,7 +21,23 @@ const references = async (DTO, uniqueObj) => {
         const recordType = DTO[current].type.constructor.name;
         let numberType = false;
         modelToQuery = DTO[current].references;
-        const modelName = modelToQuery.model;
+        
+        // all the ways a reference can be defined in Sequelize https://sequelize.org/api/v6/class/src/model.js~model#static-method-init
+        let modelName;
+        if (typeof modelToQuery === 'string') {
+          // references: string
+          modelName = modelToQuery;
+        } else if (modelToQuery.model && typeof modelToQuery.model === 'string') {
```

Comment:
Should we use optional chaining here to remove the unneeded modelToQuery.model in each of the if statements?

Reply:
I tried this out and tested it, it worked well so I've pushed up the change